documentation/ProjectReference-Protocol.md

```diff
@@ -4,6 +4,10 @@ The MSBuild engine doesn't have a notion of a “project reference”—it only
 
 That's a powerful tool, but no one would want to have to specify how to build every single reference in every single project. The common targets introduce an item, `ProjectReference`, and a default process for building references declared via that item.
 
+Protocol implementation:
```

Comment:
Say "Default protocol implementation" since it's possible to do yourself (even if it's a hassle).